### PR TITLE
Add limit and offset in getVoters API

### PR DESCRIPTION
**Pull request by navinpurohit**
Change in getVoters API to handle limit and offset to handle large data from an explorer API call. Rather than showing complete Voters list for Delegate on the Delegate info page, show only limited data and to see more, click on "Load more...".

### CODE REVIEW

File: sql/delegates.js [hunk 1]
```diff
@@ -50,7 +50,9 @@ let DelegatesSql = {
 
 	insertFork: 'INSERT INTO forks_stat ("delegatePublicKey", "blockTimestamp", "blockId", "blockHeight", "previousBlock", "cause") VALUES (${delegatePublicKey}, ${blockTimestamp}, ${blockId}, ${blockHeight}, ${previousBlock}, ${cause});',
 
-	getVoters: 'SELECT ARRAY_AGG("accountId") AS "accountIds" FROM mem_accounts2delegates WHERE "dependentId" = ${publicKey}',
+	getVoters: 'SELECT accounts.address, accounts.balance, encode(accounts."publicKey", \'hex\') AS "publicKey" FROM mem_accounts2delegates delegates INNER JOIN mem_accounts accounts ON delegates."accountId" = accounts.address WHERE delegates."dependentId" = ${publicKey} ORDER BY accounts."balance" DESC LIMIT ${limit} OFFSET ${offset}',
```
Comment (Verbalist) [inline sql/delegates.js]: It`s a bad request to db. Please optimise it.

Comment (Verbalist) [reply sql/delegates.js]: ORDER BY accounts."balance"

**Comment by navinpurohit** [comment]
Right now what you want to change in the above query as this API call is far better than the previous one. This query only calls from explorer when we click delegate info link. Please explain in detail. Thanks

**Comment by Verbalist** [comment]
ORDER BY accounts."balance" balance it`s bad field for order. Use only indexed fields

**Comment by Verbalist** [comment]
All new pull request to master

**Pull request closed without merging**